packages/sdk/src/Config.ts

```diff
@@ -0,0 +1,64 @@
+import { BaseProvider } from '@ethersproject/providers';
+import {
+  Environment,
+  ConfigDomains,
+  EthereumConfig,
+  SubstrateConfig,
+  Resource,
+} from 'types/config';
+
+export class Config {
+  private devnetConfigUrl = 'https://config.develop.buildwithsygma.com/share/';
```

Comment:
Lets put URLs to some constants file as constant.

Reply:
If we use the `sdk` from the browser, we are going to have issues with the redirection to fetch the `shared-config` resource. This is because ABL rules don't allow you to setup headers `cors`. Quick solution to this is to put the cloudflare url until we move forward with the solution proposed by @eedygreen

Reply:
sorry, redaction error, ALB: Aplication load balancer.